static/js/lib/base64.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
I didn't remember the snippet there being so simple. I've switched to using that and left a comment.